README.md

```diff
@@ -150,8 +150,33 @@ Please use GitHub’s issue tracker and pull request features.
 
 2. Install: `npm install`
 
-3. Use `npm run serve` to launch a local instance of the service. This will load
-   data from Cloud Storage; data can take up to a minute to be loaded and ready.
+3. Launch the local server:
+
+```zsh
+mkdir -p data/json
+```
+
+Then, either:
+
+   1. Copy the latest data:
+
+```zsh
+mkdir -p data/json
+cd data/json
+curl https://storage.googleapis.com/web-api-confluence-data-cache/latest/json/org.chromium.apis.web.ApiVelocityData.json > org.chromium.apis.web.ApiVelocityData.json
+curl https://storage.googleapis.com/web-api-confluence-data-cache/latest/json/org.chromium.apis.web.ReleaseWebInterfaceJunction.json > org.chromium.apis.web.ReleaseWebInterfaceJunction.json
+curl https://storage.googleapis.com/web-api-confluence-data-cache/latest/json/org.chromium.apis.web.BrowserMetricData.json > org.chromium.apis.web.BrowserMetricData.json
+curl https://storage.googleapis.com/web-api-confluence-data-cache/latest/json/org.chromium.apis.web.WebInterface.json > org.chromium.apis.web.WebInterface.json
+curl https://storage.googleapis.com/web-api-confluence-data-cache/latest/json/org.chromium.apis.web.Release.json > org.chromium.apis.web.Release.json
+cd ../..
+```
+
+or
+
+   2. [Collect the data yourself](#collecting-data).
+
+Finally, use `npm run serve` to launch a local instance of the service. This
```

Comment:
Does this step replace "To run the service locally based on your generated data invoke", or is`node main/serve.js "LOCAL" "DEV"` different from what `npm run serve` does? If it is different, fold this and also step 4 into the "Copy the latest data" branch?

Reply:
This step will invoke `scripts/serve.js "LOCAL" "DEV"` (while also running `webpack --watch` in parallel).

Reply:
I see. Can it be just npm run serve everywhere? Doesn't matter much.

Reply:
Maybe I'm forgetting context from offline discussion. The README doesn't manually invoke `serve.js`, so the only instructions here are `npm run serve` "everywhere".